src/hooks/useGoogleBooksApi.ts

```diff
@@ -0,0 +1,23 @@
+import { GoogleBook } from '../api/GoogleBook'
+import { useEffect, useState } from 'react'
+export const useGoogleBooksApi = <T>(path: string) => {
+  const [response, setResponse] = useState<T>()
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [searchTerm, doFetch] = useState<string>(path)
+  const [isError, setIsError] = useState<boolean>(false)
```

Comment:
loading, error instead of isLoading is more consistent with the other custom hooks